core/services/ocrcommon/data_source.go

```diff
@@ -85,7 +94,7 @@ func NewInMemoryDataSource(pr pipeline.Runner, jb job.Job, spec pipeline.Spec, l
 	}
 }
 
-var _ ocrtypes.DataSource = (*dataSource)(nil)
+//var _ ocr1types.DataSource = (*dataSource)(nil)
```

Comment:
Remove or restore?

Reply:
TBH, I don't get why that is there, commented it to see if anything breaks and forgot about it. I guess we can remove it?

Reply:
It verifies that `*dataSource` conforms to `ocr1types.DataSource`. Keeping it makes that explicit.

Reply:
Restored in [678a530 (#8959)](https://github.com/smartcontractkit/chainlink/pull/8959/commits/678a53083c1fdab70f274c761a44c0956decb8f4)